zimui/src/components/TopicHome.vue

```diff
@@ -88,7 +93,11 @@ const goToPreviousPage = () => {
 </script>
 
 <template>
-  <div v-if="topic" class="content">
+  <div v-if="main.errorMessage" class="error-container">
+    <img :src="errimage" class="error-image" />
```

Comment:
Why are you using a ref? `src="../assets/Error.jpg"` is not sufficient to do the job?

Reply:
I was trying to maintain consistency in the codebase, the image was imported this way in the `TopicCard.vue`.